sound/soc/amd/acp/acp-sdw-sof-mach.c

```diff
@@ -155,6 +155,7 @@ static int create_sdw_dailink(struct snd_soc_card *card,
 		int num_codecs = sof_dai->num_devs[stream];
 		int playback, capture;
 		int j = 0;
+		int no_pcm = 1;
```

Comment:
We can just pass 1 to `asoc_sdw_init_dai_link`. No need to add a new variable.

Reply:
Agreed. Will make the change.